docs/Systems/LayoutAndSpacing/LayoutAndSpacing.stories.mdx

```diff
@@ -0,0 +1,43 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+import { CATEGORIES } from "../../../storybook/constants"
+import { SpacingTokens, MarginExample } from "./DemoComponents"
+
+<Meta title={`${CATEGORIES.systems}/Layout and Spacing`} />
+
+# Layout and Spacing
+
+<br/>
+
+## Naming convention
+
+Kaizen's spacing tokens use pixel values to reference equivalent rem values. For example, the spacing token `16` refers to `1rem`, because `16px` = `1rem`.
+
+<SpacingTokens/>
+
+<br/>
+
+## Reasoning
+
+Engineers need to use `rem` when spacing for accessibility reasons. However, Figma does not allow `rem` values when creating designs. This approach allows engineers to recieve pixel values from a designer, and write code in terms of pixel values without having to calculate what the corresponding `rem` value would be.
+
+<br/>
+
+## Implementation
+
+Kaizen's spacing tokens are easily available through our Tailwind package <LINK_NEEDED>.
```

Comment:
Link this to the Getting Started page in the Tailwind section?

Reply:
Or would this be better to link to a spacing story where we list out the spacing values in a table?  If so I think we could add that in a separate commit, then update this link